### ISSUE TITLE
Prepare evaluation metrics (f1 score)

### ISSUE THREAD

**Issue by jdieguvi15** [45truc/NER_task#5]
habria quue usar distintas metricas como accuracy o recall e importante usar también weighted F1 score ! (lo pide)

weighted entiendo que se refiere a que pongamos el peso de las que son clase 0 (not a named entity) lo pongamos a 0 y el resto weighted en función del suport (creo que se puede hacer fácil con [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)
